marlowe-playground-client/src/Demos/State.purs

```diff
@@ -0,0 +1,52 @@
+module Demos.State where
+
+import Prelude hiding (div)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (wrap)
+import Demos.Types (Action(..), Demo)
+import Effect.Aff.Class (class MonadAff)
+import Halogen (ClassName(..), ComponentHTML, HalogenM)
+import Halogen.HTML (HTML, button, div, h1_, h2_, hr_, span, text)
+import Halogen.HTML.Events (onClick)
+import Halogen.HTML.Properties (class_, classes)
+import Marlowe (SPParams_)
+import Projects.Types (Lang(..))
+import Servant.PureScript.Settings (SPSettings_)
+import Types (ChildSlots)
+
+handleAction ::
+  forall m state.
+  MonadAff m =>
+  SPSettings_ SPParams_ ->
+  Action -> HalogenM state Action ChildSlots Void m Unit
+handleAction _ _ = pure unit
```

Comment:
What's going on here then? 🤔

Reply:
I created a nice MVC for demos but it doesn't need the C bit. I should probably change this to View.purs and get rid of the handleAction?

Reply:
best to look at the demos code again after my most recent push